Announcements.md

```diff
@@ -0,0 +1,15 @@
+# Announcements
+
+## Thursday, April 1, 2021
+
+[Azure Private Link for Azure Cache for Redis](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-private-link) is now GA!
+
+Use Azure Private Link to connect to an Azure Cache for Redis instance from your virtual network via a private endpoint to eliminate data exposure to the public internet.
+
+## Tuesday, March 2, 2021
+
+The [Enterprise and Enterprise Flash tiers](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-overview#service-tiers) of Azure Cache for Redis are now GA! This includes the [preview active geo-replication](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-how-to-active-geo-replication), with up to 99.999% availability.
+
+## Wednesday, February 3, 2021
+
+Today, [Azure Event Grid is now in public preview for Azure Cache for Redis](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-event-grid)!
```

Comment:
Also include public preview announcements for Zone Redundancy and Redis 6.0? What do you think about having a section/page devoted to features that are in preview?

Reply:
That's a good thought. Hmmm. Maybe it'd work well to have a "pinned" section at the top for features in preview that haven't yet been promoted to GA. I'll experiment with that a little.